pycode/memilio-simulation/memilio/simulation/abm.cpp

```diff
@@ -200,9 +187,10 @@ PYBIND11_MODULE(_simulation_abm, m)
     py::class_<mio::abm::World>(m, "World")
         .def(py::init<mio::abm::GlobalInfectionParameters>(),
              py::arg("infection_parameters") = mio::abm::GlobalInfectionParameters{})
-        .def("add_location", &mio::abm::World::add_location, py::arg("location_type"), py::arg("num_cells") = 0,
+        .def("add_location", &mio::abm::World::add_location, py::arg("location_type"), py::arg("num_cells") = 1,
+             py::return_value_policy::reference_internal)
```

Comment:
Is this correct? `add_location ` does not return a reference (in contrast to `add_person`)

Reply:
Good Call!